### PR TITLE
Update to react-navigation 3.0.0

### LINKED ISSUE #1342: Update to React Navigation 3.0

### PR DESCRIPTION
**Pull request by ferrannp**
Fixes #1342.

- [x] Add gesture handler native module in Gitlab for Android
- [x] Make sure playgrounds behave correctly

### CODE REVIEW

File: android/app/src/main/java/com/reactnativeapp/MainApplication.java [hunk 1]
```diff
@@ -38,6 +39,7 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
             return Arrays.asList(
                     new MainReactPackage(),
+            new RNGestureHandlerPackage(),
```
Comment (tbergq) [inline android/app/src/main/java/com/reactnativeapp/MainApplication.java]: please align this with the rest, my OCD is ticking

Comment (ferrannp) [reply android/app/src/main/java/com/reactnativeapp/MainApplication.java]: Haha I will, but that is `react-native link` doing it :)

Comment (tbergq) [reply android/app/src/main/java/com/reactnativeapp/MainApplication.java]: I know, but I always go in and add these extra tabs to this file 🙈

**Comment by ferrannp** [comment]
Please @tbergq some testing of routes and double check.

**Comment by ferrannp** [comment]
I'm off today @tberg but you can go for it, `rnkimobike` to 0.0.35 and bump code push.

**Pull request merged**